fixed for error "undefined reference to `vtable for Phonon::ObjectDescriptionModel<(Phonon::ObjectDescriptionType)0>'"
that occurs (only) on some systems (e.g. RHEL6/Sandy Bridge)
based on https://forum.qt.io/topic/35044/build-failure-4-8-5-on-mac-with-phonon/9
author: Kenneth Hoste (HPC-UGent)
--- qt-everywhere-opensource-src-4.8.6/src/3rdparty/phonon/phonon/objectdescriptionmodel.h.orig	2015-06-22 23:18:54.723573581 +0300
+++ qt-everywhere-opensource-src-4.8.6/src/3rdparty/phonon/phonon/objectdescriptionmodel.h	2015-06-22 23:19:16.902399019 +0300
@@ -141,7 +141,7 @@
 
 /* Required to ensure template class vtables are exported on both symbian
 and existing builds. */
-#if (defined(Q_OS_SYMBIAN) && defined(Q_CC_RVCT)) || defined(Q_CC_CLANG)
+#if (defined(Q_OS_SYMBIAN) && defined(Q_CC_RVCT)) || defined(Q_CC_CLANG) || defined(Q_CC_INTEL)
 // RVCT compiler (2.2.686) requires the export declaration to be on the class to export vtables
 // MWC compiler works both ways
 // GCCE compiler is unknown (it can't compile QtCore yet)
